modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/exportimport/portlet/preferences/processor/DDMFormExportImportPortletPreferencesProcessor.java

```diff
@@ -90,6 +93,39 @@ public PortletPreferences processExportPortletPreferences(
 			return portletPreferences;
 		}
 
+		long groupId = GetterUtil.getLong(
+			portletPreferences.getValue("groupId", null));
+
+		if (groupId <= 0) {
+			if (_log.isWarnEnabled()) {
+				_log.warn(
+					"No group ID found in preferences of portlet " + portletId);
+			}
+
+			return portletPreferences;
+		}
+
+		Group group = _groupLocalService.fetchGroup(groupId);
+
+		if (group == null) {
+			if (_log.isDebugEnabled()) {
+				_log.debug("No group found with group ID " + groupId);
+			}
+
+			return portletPreferences;
+		}
+
+		if (ExportImportThreadLocal.isStagingInProcess() &&
+			!group.isStagedPortlet(
+				DDMPortletKeys.DYNAMIC_DATA_MAPPING_FORM_ADMIN)) {
+
+			if (_log.isDebugEnabled()) {
+				_log.debug("Form is not staged in the site " + group.getName());
+			}
+
+			return portletPreferences;
+		}
```

Comment:
@vendeltoreki, can you please explain to me why did you prefer to log the errors on debug level instead of warn level?

Reply:
@natocesarrego , this is the same as before in [JournalContentExportImportPortletPreferencesProcessor](https://github.com/liferay/liferay-portal/blob/b3173da81b62430f7150b695d5e944b224279580/modules/apps/journal/journal-content-web/src/main/java/com/liferay/journal/content/web/internal/exportimport/portlet/preferences/processor/JournalContentExportImportPortletPreferencesProcessor.java#L124-L155). The "No group found" log message can be a warning, since incorrect groupIds can mean that the data is corrupt.
However, the Forms portlet not being staged in a staged environment is completely normal (that's the default setting). I wouldn't change that to a warning.

Thanks for the review! :)

Reply:
Makes sense to me, Vendel. Thanks.